src/test/java/org/jabref/logic/integrity/NoBibTexFieldCheckerTest.java

```diff
@@ -2,99 +2,62 @@
 
 import java.util.Collections;
 import java.util.List;
+import java.util.stream.Stream;
 
 import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
 import org.jabref.model.entry.field.StandardField;
 import org.jabref.model.entry.field.UnknownField;
 
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class NoBibTexFieldCheckerTest {
 
     private final NoBibtexFieldChecker checker = new NoBibtexFieldChecker();
 
-    @Test
-    void abstractIsNotRecognizedAsBiblatexOnlyField() {
-        BibEntry entry = new BibEntry();
-        entry.setField(StandardField.ABSTRACT, "test");
-        assertEquals(Collections.emptyList(), checker.check(entry));
-    }
-
-    @Test
-    void addressIsNotRecognizedAsBiblatexOnlyField() {
-        BibEntry entry = new BibEntry();
-        entry.setField(StandardField.ADDRESS, "test");
-        assertEquals(Collections.emptyList(), checker.check(entry));
-    }
-
-    @Test
-    void afterwordIsRecognizedAsBiblatexOnlyField() {
-        BibEntry entry = new BibEntry();
-        entry.setField(StandardField.AFTERWORD, "test");
-        IntegrityMessage message = new IntegrityMessage("biblatex field only", entry, StandardField.AFTERWORD);
+    private static Stream<Arguments> nonBiblatexOnlyFields() {
+        return Stream.of(
+                // arbitrary field
+                Arguments.of(new UnknownField("fieldNameNotDefinedInThebiblatexManual"), Collections.emptyList()),
+
+                // these fields are displayed by JabRef as default
+                Arguments.of(StandardField.ABSTRACT, Collections.emptyList()),
+                Arguments.of(StandardField.COMMENT, Collections.emptyList()),
+                Arguments.of(StandardField.DOI, Collections.emptyList()),
+                Arguments.of(StandardField.URL, Collections.emptyList()),
+
+                // these fields are not recognized as biblatex only fields
+                Arguments.of(StandardField.ADDRESS, Collections.emptyList()),
+                Arguments.of(StandardField.INSTITUTION, Collections.emptyList()),
+                Arguments.of(StandardField.JOURNAL, Collections.emptyList()),
+                Arguments.of(StandardField.KEYWORDS, Collections.emptyList()),
+                Arguments.of(StandardField.REVIEW, Collections.emptyList())
```

Comment:
Each thing gets `, Collections.emptyList()` as paramter. Thus, this can be removed.

Maybe, even `Stream.of(List.of(StandardField.ABSTRACT, StandardField.COMMENT, ...)` can be used.

```suggestion
                Arguments.of(new UnknownField("fieldNameNotDefinedInThebiblatexManual")),

                // these fields are displayed by JabRef as default
                Arguments.of(StandardField.ABSTRACT),
                Arguments.of(StandardField.COMMENT),
                Arguments.of(StandardField.DOI),
                Arguments.of(StandardField.URL),

                // these fields are not recognized as biblatex only fields
                Arguments.of(StandardField.ADDRESS),
                Arguments.of(StandardField.INSTITUTION),
                Arguments.of(StandardField.JOURNAL),
                Arguments.of(StandardField.KEYWORDS),
                Arguments.of(StandardField.REVIEW)
```

Reply:
Code is updated as suggested. :)